src/App.css

```diff
@@ -1,6 +1,10 @@
 .App {
   color: #404040;
   text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content:flex-end;
+  min-height:100vh;
```

Comment:
nit: space between justify-content and flex-end. Same for min-height

Reply:
done